testplan/runners/pools/remote.py

```diff
@@ -74,6 +74,10 @@ def __init__(self, **options) -> None:
             options["pool_type"] = "thread"
         super().__init__(**options)
 
+    @property
+    def host(self) -> str:
```

Comment:
Perhaps name it `remote_host`? No strong opinion here, up to your decision, just suggesting.

Reply:
Local pool uses this attribute, too